### ISSUE TITLE
Recommend methods does not provide parameter for RecommendStrategy

### ISSUE THREAD

**Issue by VaishaliD2** [qdrant/qdrant-dotnet#54]
Below is how RecommedAsync method looks like-
```
public async Task<IReadOnlyList<ScoredPoint>> RecommendAsync(
		string collectionName,
		IReadOnlyList<PointId> positive,
		IReadOnlyList<PointId>? negative = null,
		ReadOnlyMemory<Vector>? positiveVectors = null,
		ReadOnlyMemory<Vector>? negativeVectors = null,
		Filter? filter = null,
		SearchParams? searchParams = null,
		ulong limit = 10,
		ulong offset = 0,
		WithPayloadSelector? payloadSelector = null,
		WithVectorsSelector? vectorsSelector = null,
		float? scoreThreshold = null,
		string? usingVector = null,
		LookupLocation? lookupFrom = null,
		ReadConsistency? readConsistency = null,
		ShardKeySelector? shardKeySelector = null,
		TimeSpan? timeout = null,
		CancellationToken cancellationToken = default)
```

There is no option to specify RecommendStrategy in any of the RecommendMethods available in this QdrantClient class.

**Comment by VaishaliD2** [comment]
I opened this issue and would like to take it up myself. 

If there are any specific guidelines or additional information I should be aware of, please let me know.
Thanks

**Comment by Anush008** [comment]
Resolved with #55
